SU-HSDO/suhumsci
# ARCH-382 Disable the page title block on node 404 page

# READY FOR REVIEW

# Summary
- Disable the page title block on 404 page.
- Failed test should be solved in https://github.com/SU-HSDO/suhumsci/pull/97

# Needed By (Date)
- End of sprint sept 27

# Urgency
- medium

# Steps to Test
1. Checkout this branch
1. sync `blt drupal:sync --site=archaeology --partial`
1. visit any page that produces a 404 response
1. ensure the page title "Page Not Found" only displays 1 time.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
